tests/linux/test_backend_utils.cpp

```diff
@@ -129,25 +100,14 @@ void simulate_qemuimg_convert(const mpt::MockProcess* process, const QString& im
 
 template <class Matcher>
 void test_image_resizing(const char* img, const mp::MemorySize& img_virtual_size, const mp::MemorySize& requested_size,
-                         const char* qemuimg_info_output, const mp::ProcessState& qemuimg_info_result,
-                         bool attempt_resize, const mp::ProcessState& qemuimg_resize_result,
-                         mp::optional<Matcher> throw_msg_matcher)
+                         const mp::ProcessState& qemuimg_resize_result, mp::optional<Matcher> throw_msg_matcher)
 {
     auto process_count = 0;
     auto mock_factory_scope = mpt::MockProcessFactory::Inject();
-    const auto expected_final_process_count = attempt_resize ? 2 : 1;
 
     mock_factory_scope->register_callback([&](mpt::MockProcess* process) {
-        ASSERT_LE(++process_count, expected_final_process_count);
+        ASSERT_LE(++process_count, 1);
         if (process_count == 1)
-        {
-            auto msg = QByteArray{qemuimg_info_output};
-            if (msg.isEmpty())
-                msg = fake_img_info(img_virtual_size);
-
-            simulate_qemuimg_info(process, img, qemuimg_info_result, msg);
-        }
-        else
         {
             simulate_qemuimg_resize(process, img, requested_size, qemuimg_resize_result);
         }
```

Comment:
The `if` is spurious with the `ASSERT_LE` above.

Reply:
Right, removed.